postcss.config.js

```diff
@@ -1,16 +0,0 @@
-const autoprefixer = require('autoprefixer');
-
-module.exports = {
-    plugins: [
-        autoprefixer({
-            browsers: [
-                'last 3 Chrome versions',
-                'last 3 Firefox versions',
-                'Explorer >= 11',
-                'last 2 Edge versions',
-                'last 3 Opera versions',
-                'Safari >= 8',
-            ],
-        }),
-    ],
-};

```

Comment:
Why remove this configuration?

Reply:
The `browsers` option of the autoprefixer is deprecated, it now automatically uses the `browserslist` config we have in the [package.json](https://github.com/coveo/react-vapor/blob/master/package.json#L34-L39)